app/models/manageiq/providers/workflows/automation_manager/configuration_script_source.rb

```diff
@@ -1,2 +1,36 @@
 class ManageIQ::Providers::Workflows::AutomationManager::ConfigurationScriptSource < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptSource
+  FRIENDLY_NAME = "Embedded Workflows Repository".freeze
+
+  def self.display_name(number = 1)
+    n_('Repository (Embedded Workflows)', 'Repositories (Embedded Workflows)', number)
+  end
+
+  def sync
+    update!(:status => "running")
+
+    transaction do
+      current = configuration_script_payloads.index_by(&:name)
+
+      git_repository.update_repo
+      git_repository.with_worktree do |worktree|
+        worktree.ref = scm_branch
+        worktree.blob_list.each do |filename|
+          next if filename.start_with?(".") || !filename.end_with?(".asl")
```

Comment:
So, something like `.example.asl` will be skipped?  Curious why the `.` is necessary to check?

Reply:
Yes, this is to skip hidden files/folders.

For compatibility with EmbeddedAnsible: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb#L107 (they also have specs for this case)